tests/ut-gamemaker.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import gamemaker
+import database
+import team
+import user
+
+class TestMakerLogin(unittest.TestCase):
+  def set_up(self):
+    self.database = database.Database()
+    self.maker1 = gamemaker.GameMaker()
+  
+  def test_bad_login(self):
+     self.assertEquals(self.maker1.login("Notvalidusername","Notvalidpassword"), False, "Bad response to invalid login")
+     self.assertEquals(self.database.get_current_user(), {}, "Current user is not null")
+  def test_good_login(self):
+     self.assertEquals(self.maker1.login("maker","password"), True, "Maker should have logged in")
+     self.assertEquals(self.database.get_current_user(), {"maker" : "password"}, "Current user should be maker")
```

Comment:
Not sure yet on the implementation of this. current_user will generally hold an object of type Team, but obviously it won't hold an object of type GameMaker. My guess is that we'll create some other GM class that just has username and password fields, that way it will still work with out implementation using current_user